ext/ruby_newt/ruby_newt.c

```diff
@@ -1480,6 +1500,8 @@ void Init_ruby_newt(){
   cWidget = rb_define_class_under(mNewt, "Widget", rb_cObject);
   rb_define_method(cWidget, "callback", rb_ext_Widget_callback, -1);
   rb_define_method(cWidget, "takes_focus", rb_ext_Widget_takesFocus, 1);
+  rb_define_method(cWidget, "get_position", rb_ext_Widget_GetPosition, 0);
+  rb_define_method(cWidget, "get_size", rb_ext_Widget_GetSize, 0);
```

Comment:
Allright, I see there is a commit down below named "Add tests" so I hope it's all there. :-)